docs/docs/guides/wagmi_usage/wagmi.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 1
+sidebar_label: 'Wagmi Web3js Adaptor'
+title: 'Wagmi Web3js Adaptor'
+---
+
+If you're using [Wagmi](https://wagmi.sh/react/getting-started#use-wagmi) and want to add web3.js, use this provider in your project:
+
+
+```typescript
```

Comment:
and for https://github.com/web3/web3.js/issues/6746 there should be signer adapter doc as well